script/cli/check.lua

```diff
@@ -8,7 +8,7 @@ local util       = require 'utility'
 
 local numThreads = tonumber(NUM_THREADS or 1)
 
-local exe = arg[-1]
```

Comment:
Strange enough 😕 on windows and macos, `arg[-1]` works perfectly while `arg[-2]` is `nil`

Moreover I've been using **LuaLS** in my projects github action lint workflow, which uses the **ubuntu-latest** vm and it works normally as well. The gha step cmd that I used is simply:
```yml
      - name: Run LuaLS
        run: |
          ${LUALS_PATH}/bin/lua-language-server --check=. --num_threads=2 --checklevel=Error
```

---
Would the issue that you encountered is environment specific?
- Which shell did you use?
- Did you use symlink for the `lua-language-server` binary, or use some custom launch script?

Reply:
I use archlinux's lua-language-server. shell is zsh.

Reply:
Now it should work for all machines.